lib/mintel/dashboards/components/panels/common.libsonnet

```diff
@@ -97,19 +118,22 @@ local statusdotsPanel = import 'statusdots_panel.libsonnet';
       title=title,
       description=description,
       colors=colors,
+      colorValue=colorValue,
+      decimals=decimals,
       thresholds=thresholds,
       format=format,
       gaugeShow=true,
       gaugeMinValue=gaugeMinValue,
       gaugeMaxValue=gaugeMaxValue,
       height=height,
       valueFontSize=valueFontSize,
+      valueName=valueName,
       transparent=transparent,
       interval=interval,
       postfix=postfix,
       span=span,
     )
-    .addTarget(promQuery.target(query, instant)),
+    .addTarget(promQuery.target(query, interval=interval, intervalFactor=intervalFactor)),
```

Comment:
why are we changing `instant` to `interval` ?

Reply:
Instant is being removed as a parameter passed to promQuery as it was actually setting the format field incorrectly, interval is a new parameter passed.

Reply:
for this specific widget though we want that interval to be "instant" so that it only shows the current value rather than the one from the time window selected.

is it happening ?

Reply:
Resolved by setting instant correctly.  Previously the value intended for instant was being set to format.